### PR TITLE
Feature/login-styles

### PR DESCRIPTION
**Pull request by condef5**
### This pull request includes:

* Add components for login
* Add styles for each component
* Add basic logic to render logic or app

### CODE REVIEW

File: client/src/Login/GroupImage.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    position: relative;
+    width: 100px;
+    height: 100px;
+    margin-bottom: 16px;
+  `;
+
+  const image = css`
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
```
Comment (CesarCachay) [inline client/src/Login/GroupImage.js]: Great! this line allow us to fit the size of the image in the container! 
More information here: https://www.w3schools.com/css/css3_object-fit.asp

File: client/src/Login/GroupActivity.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+const success = css`
+  ${info}
+  &:before {
+    background-color: #43b581;
+  }
+`;
+
+const nothing = css`
+  ${info}
```
Comment (MariethPerez) [inline client/src/Login/GroupActivity.js]: PLease,Could you explain me Why you use '$'?

File: client/src/index.js [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+function Index() {
+  const isLoggin = true;
+  if (isLoggin) {
+    return <App />;
+  }
+  return <Login />;
+}
+
+const $root = document.getElementById('root');
+render(<Index />, $root);
```
Comment (MariethPerez) [inline client/src/index.js]: Good job :)  Thanks for your colaboration.

File: client/src/Login/Form.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  color: #fff;
+  border-radius: 5px;
+  background-color: #516dd2;
+  cursor: pointer;
+`;
+
+function Login() {
+  return (
+    <form className={Form}>
+      <span className={username}>USERNAME</span>
```
Comment (stg101) [inline client/src/Login/Form.js]: Can you explain to me why you have used `className` instead of `css`

**Pull request merged**